### ISSUE TITLE
Tap to Place inspector issue (advanced properties)

### ISSUE THREAD

**Issue by ltwlf** [microsoft/MixedRealityToolkit-Unity#9645]
## Describe the bug

When you open the Advanced Property section of "TapToPlace" you get the following exception and the Magnetic Surfaces setting is missing:
```
NullReferenceException: Object reference not set to an instance of an object
UnityEditor.EditorGUILayout.IsChildrenIncluded (UnityEditor.SerializedProperty prop) (at <c9b5231e365148a4bc210cc4e439206d>:0)
UnityEditor.EditorGUILayout.PropertyField (UnityEditor.SerializedProperty property, UnityEngine.GUILayoutOption[] options) (at <c9b5231e365148a4bc210cc4e439206d>:0)
Microsoft.MixedReality.Toolkit.Utilities.Editor.Solvers.TapToPlaceInspector.RenderAdvancedProperties () (at Library/PackageCache/com.microsoft.mixedreality.toolkit.foundation@98717fc5a800-1618067337299/SDK/Editor/Inspectors/Utilities/Solvers/TapToPlaceInspector.cs:117
```

This issue is also described here:
https://techcommunity.microsoft.com/t5/mixed-reality-feedback-and-help/error-when-trying-to-open-advanced-properties-section-of-tap-to/m-p/2202628

The issue seems to be caused by orphaned serialized properties in the inspector script "TapToPlaceInspector". 

## To reproduce

1. Add TapToPlace to a Game Object
2. Open Advanced Settings in the inspector

## Expected behavior
No exception should be thrown and Magnetic Surfaces setting should be visible

## Screenshots

![image](https://user-images.githubusercontent.com/965766/114294753-ead1e080-9aa0-11eb-8676-27fa31ed9bb6.png)

## Your setup (please complete the following information)

- Unity Version 2020.3.3f1
- MRTK Version v2.7.0 Preview 2]

## Target platform (please complete the following information)

- Editor

**Comment by ltwlf** [comment]
I'll create a pull request to fix the issue.

**Comment by keveleigh** [comment]
Fixed by #9646. Thanks for the contribution! 🎉

**Comment by fschmidmichels** [comment]
Hi! Will this also be fixed in a 2.6.x release? Or do I have to update to 2.7 beta version? Thanks

**Comment by keveleigh** [comment]
Pinging @polar-kev to provide an update. I don't believe we're planning on shipping another 2.6.x release but are working towards 2.7's release next month / early June.
